packages/ui/src/components/va-switch/VaSwitch.stories.ts

```diff
@@ -1,12 +1,51 @@
+import { userEvent } from '@storybook/testing-library'
+import { StoryFn } from '@storybook/vue3'
 import { defineComponent } from 'vue'
-import VaSwitch from './VaSwitch.demo.vue'
+import VaSwitchDemo from './VaSwitch.demo.vue'
+import VaSwitch from './VaSwitch.vue'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaSwitch',
   component: VaSwitch,
 }
 
-export const Default = defineComponent({
-  components: { VaSwitch },
+export const Default = () => defineComponent({
+  components: { VaSwitch: VaSwitchDemo },
   template: '<VaSwitch/>',
 })
+
+export const ChangeEvent: StoryFn = () => ({
+  components: { VaSwitch },
+
+  data: () => ({ value: false, changeCount: 0 }),
+
+  methods: {
+    onChange (e) {
+      if (e.target) {
+        this.changeCount++
+      }
+    },
+  },
+
+  template: `
+    [changes count]: {{ changeCount }}
```

Comment:
small thing, but the convention is something like in Backtop > ClickEvent.

```html
<p>count: <span data-testid="count">{{ count }}</span></p>
```

```typescript
(canvas.getByTestId('clicked').innerText).toBe('2')
```

Reply:
querySelector seems much more clean solution for me, than within and getByTestId. I'd prefer querySelector.